frontend/src/modules/EditZing/utils/emailTemplates.tsx

```diff
@@ -0,0 +1,252 @@
+//     other students. We are so glad you reached out! Studying with peers is
+//     known to be an effective learning tool, but whether you are learning online or
+//     in-person it can be challenging, especially in large classes, to find study
+//     partners.
+//     </div>
+//     <br>
+//     <div>
+//     <b> ${newStudent} </b>, within the past few days we tried to match
+//     you with another student who requested a study partner for the same
+//     class(es) you listed but haven’t had any luck. Fortunately, a small group that
+//     we matched recently is happy to have another student join them in studying.
+//     Thank you<b>${students}</b>!
+//     </div>
+//     <br>
+//     <div>
+//     (Due to privacy rules we are not able to disclose which classes the student(s)
+//     receiving this email are taking, but you have been matched because you
+//     expressed interest in finding study partners for the same course.)
+//     </div>
+//     <br>
+//     <div>
+//     Whether you will be studying together in -person or online, use the LSC’s tips
+//     for setting your group’s agenda and making the most out of studying
+//     together.Consider working together at a time when you can use course
+//     office hours or LSC tutoring to answer questions that may arise!
+//     </div>
+//     <br>
+//     <div>
+//     If you’d like more info about studying together please contact LSC Study
+//     Partners lscstudypartners@cornell.edu  We will be following up later in the
+//     semester to find out how it’s going.
+//     </div>
+//     <br>
+//     <br>
+//     Happy studying!`
+// }
+
+const checkIn = () => {
+  return ` 
+    <div>  Dear students, </div>
+      <br> 
+      <br> 
+    <div> 
+      According to our records we matched you as study partners for a class you 
+      requested other students to study with. 
+    </div>  
+      <br> 
+    <div> 
+      We are writing to check in. We hope your group has been able to meet and 
+      that you are finding it useful to study with others! Please keep in mind that 
+      LSC peer study skills tutors are available to help your group set up a 
+      constructive group work environment-- you can self-enroll in the study skills 
+      tutors’ Canvas site to see when drop-in hours are held, you can show up any 
+      time that works for your group.
+    </div>
+      <br> 
+    <div> 
+      We know that sometimes matched students are not able to find a time to 
+      meet that works for everyone, or one person is auditing and decides not to 
+      do group work after all, or someone drops the class, and so on and so forth. 
+      If you need to be re-grouped please DO NOT fill out the webform again. Just 
+      reply to this email to let us know what’s going on, and we’ll try to help.
+    </div>
+      <br> 
+      <br> 
+    <div> 
+      Take care, 
+      The LSC Staff
+    </div>
+    `
+}
```

Comment:
Does this file have to be named `.tsx`? As far as I can tell there's no JSX in the code and it breaks GitHub's syntax highlighting because the first character of the filename is not capitalized like a react component file